lib/Cleantalk/Custom/Cron/Cron.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
It seems Cron functionality not working - Drupal handlers used. Need to be recheck and fix

Reply:
Resolved